binaries/scoot-snapshot-db/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/scootdev/scoot/os/temp"
+	"github.com/scootdev/scoot/snapshot"
+	"github.com/scootdev/scoot/snapshot/cli"
+	"github.com/scootdev/scoot/snapshot/git/gitdb"
+	"github.com/scootdev/scoot/snapshot/git/repo"
+)
+
+func main() {
```

Comment:
There should be a help string stating that cwd must be a the git repo (even if it's implied since there's no other way to specify it)